# Stefan Teleman <stefan.teleman@oracle.com>
# Port of CilkPlus (libcilkrts) to SPARC and Solaris:
# Enable build of CilkPlus on SPARC.
# Internal patch. SPARC specific:
# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=68945
--- libcilkrts/configure.tgt	2013-10-29 11:37:47.000000000 -0700
+++ libcilkrts/configure.tgt	2015-08-05 18:18:38.002232566 -0700
@@ -37,6 +37,10 @@
     ;;
   i?86-*-*)
     ;;
+  sparc-*-*)
+    ;;
+  sparc*-*-*)
+    ;;
   *-*-*)
     UNSUPPORTED=1
     ;;
